kaldb/src/main/java/com/slack/kaldb/logstore/search/LogIndexSearcherImpl.java

```diff
@@ -175,7 +175,7 @@ private LogMessage buildLogMessage(IndexSearcher searcher, ScoreDoc hit) {
       LogWireMessage wireMessage = JsonUtil.read(s, LogWireMessage.class);
       return new LogMessage(
           wireMessage.getIndex(), wireMessage.getType(), wireMessage.id, wireMessage.source);
-    } catch (IOException e) {
+    } catch (Exception e) {
```

Comment:
We should be catching all exceptions, because some of the work potentially throws Runtime exceptions which won't get caught otherwise.